gdk4/src/rgba.rs

```diff
@@ -79,6 +79,40 @@ impl RGBA {
         }
     }
 
+    #[inline]
+    pub const fn with_red(self, red: f32) -> Self {
```

Comment:
The commit message mentions constructors but they are more of helpers functions. Also I would be demanding and ask for at least a doc comment about the use case with an example.

Reply:
Do we btw actually need an example?

Reply:
I would prefer having one yes

Reply:
It would also test that the code keep doing what it says it does

Reply:
Makes sense